### ISSUE TITLE
[Documentation] "v-navigation-drawer" in conjunction with "app" prop "disable-resize-watcher" default's to "true" ?

### ISSUE THREAD

**Issue by izznogooood** [vuetifyjs/vuetify#11104]
### Environment

**Browsers:** Chrome 80.0.3987.163
**OS:** Windows 10

### Steps to reproduce
Clone / use the complex layout and disable-resize-watcher needs to be set to false. Documentation states false is default, but not true with app prop.

Easy to figure out but a hint or note in the v-navigation-drawer documentation would be nice?

### Expected Behavior
Default "false"

### Actual Behavior
true

### Reproduction Link
<a href="https://vuetifyjs.com/en/examples/layouts/complex/" target="_blank">https://vuetifyjs.com/en/examples/layouts/complex/</a>

**Comment by MajesticPotatoe** [comment]
Can you provide some clarification?
Seems to be working as described (as in `disable-resize-watcher` is false would mean that resize-watchers are active and when you change the browser size, nav-drawer will auto open)

**Comment by izznogooood** [comment]
I'm sorry, you are absolutely right.
I got fooled by the "double negative"

/Anders

**Comment by MatthewZito** [comment]
While this issue is closed, has the Vuetify team considered setting the prop to true (i.e. "off") by default?

If you consider that this option is not-so-obvious, you'll likely have developers building route-guarded / protected nav drawers,  later discovering that on viewport resize, the watcher triggers the drawer to open - even when a user is logged out. Just seems to have some security / privacy implications to me.

In short...Not necessarily a problem with Vuetify, but it seems more intuitive to me to disable this out-of-the-box and allow developers to bind resize watchers at their discretion. I would be happy to make this change if found acceptable.

**Comment by KaelWD** [comment]
No. When the window is resized to mobile widths the drawer switches to `temporary` and closes to not interfere with the rest of the page.
